docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+# This docker-compose is for developer convenience, not for running in production.
+
+services:
+
+  postgres:
+    image: postgres:16.3
+    restart: always
+    container_name: postgres
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=hive
+      - POSTGRES_PASSWORD=hivepassword
+      - POSTGRES_DB=hive
```

Comment:
we are not planning to mount any volumes for this container?

Reply:
Not for testing purposes

Reply:
👍